cost/low_service_usage/low_service_usage.pt

```diff
@@ -2,20 +2,10 @@ name "Low Service Usage"
 rs_pt_ver 20180301
 type "policy"
 short_description "Analyze all service usage and determines recommend consolidation or deletion. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/low_service_usage/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.0"
+long_description "Version: 1.1"
 severity "low"
 category "Cost"
 
-
-parameter "param_days" do
```

Comment:
If you are removing this parameter, the readme should be updated to reflect as well as provide details about the time frame being reported on.